app/lib/ability.rb

```diff
@@ -31,6 +31,8 @@ def initialize(user, resource, controller = nil)
       global_billing_administrator_abilities(user, resource)
 
       account_administrator_abilities(user, resource, controller)
+
+      cannot :unreconcile, OrderDetail
```

Comment:
Not a bug fan of this, but for us to know whether the Facility Billing Admin can/cannot unreconcile an order detail, we need to check the facility. At the end of the day, I think we need to authorize the OrderDetail, Order or Facility, and for all of them we have a `:manage` that breaks logic. Couldn't find a better way to do this, but happy to change it to something else that feels better.

Reply:
I'm ok having a custom action for this